### PR TITLE
Added usage description for SH

### PR DESCRIPTION
**Pull request by rkobismarck**
Added usage description for SH

### CODE REVIEW

**Comment by rkobismarck** [comment]
Please help me, validating the usage function implementation. What do you think about this basic approach? do you have something else in mind?

File: AWS/instances.sh [hunk 1]
```diff
@@ -1,4 +1,12 @@
 #!/bin/bash
+
+function usage {
+    echo "usage: ./instances.sh [action] [config-file]"
+    echo "  	 [action] start,stop,state"
+    echo "  	 [config] valid route to a config file"
```
Comment (Oscarz90) [inline AWS/instances.sh]: Add here a real example of how to use it e.g.  `./instances.sh start file.config`

File: AWS/instances.py [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+import os.path
+
+
+
+def doesThisFileExist(fileName):
+
+	return os.path.isfile(os.path.dirname(os.path.realpath(__file__))+"/"+fileName)
+
+
+PARAMETER_ACTION 		= 1;
```
Comment (Oscarz90) [inline AWS/instances.py]: Move constants to the top of the file after imports

File: AWS/instances.py [hunk 3]
```diff
@@ -0,0 +1,26 @@
+#!/usr/bin/python
+import sys
+import os.path
+
+
```
Comment (Oscarz90) [inline AWS/instances.py]: Remove empty lines, only one between sections

File: AWS/instances.py [hunk 4]
```diff
@@ -0,0 +1,26 @@
...
+
+def doesThisFileExist(fileName):
+
```
Comment (Oscarz90) [inline AWS/instances.py]: The same for this line

**Review by Oscarz90 — COMMENTED** [review]
I left some recommendations, but in general everything looks fine.

**Pull request merged**